src/version/ContractVersionBase.sol

```diff
@@ -8,6 +8,6 @@ import {IVersionedContract} from "../interfaces/IVersionedContract.sol";
 contract ContractVersionBase is IVersionedContract {
     /// @notice The version of the contract
     function contractVersion() external pure override returns (string memory) {
-        return "1.3.1";
+        return "1.3.2";
```

Comment:
had to bump this to match the package version.

Reply:
👍